lms/envs/common.py

```diff
@@ -85,8 +85,6 @@
     #   by course staff.
     # .. toggle_use_cases: open_edx
     # .. toggle_creation_date: 2015-09-04
-    # .. toggle_target_removal_date: None
-    # .. toggle_warnings: None
```

Comment:
**Note**: There is a lot of these issues, and it doesn't hurt anything necessarily, but makes the docs longer and makes it harder for the useful data to stand-out. Hopefully this clean-up will help some with the contagion factor, but linting is what would really nip this in the bud.

This comment is for future discussion.